lib/driver/runtimeTool.cpp

```diff
@@ -24,211 +22,92 @@
 namespace WasmEdge {
 namespace Driver {
 
-int Tool(int Argc, const char *Argv[]) noexcept {
+int Tool(struct DriverToolOptions &Opt) noexcept {
   using namespace std::literals;
 
   std::ios::sync_with_stdio(false);
   Log::setInfoLoggingLevel();
 
-  for (const auto &Path : Plugin::Plugin::getDefaultPluginPaths()) {
```

Comment:
I found you remove the plugin loading process. Do you move this to anywhere?

Reply:
Yes, I moved it to [include/driver/tool.h](https://github.com/WasmEdge/WasmEdge/pull/2363/files#diff-5ced14ef380afec45329d6e89be838ed1a019f52f083a5ff555db60de4074741R144) line 144